core/graphelier-service/api/hndlrs/orderhandler_test.go

```diff
@@ -0,0 +1,69 @@
+package hndlrs_test
+
+import (
+	"graphelier/core/graphelier-service/models"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"graphelier/core/graphelier-service/api/hndlrs"
+	. "graphelier/core/graphelier-service/utils/test_utils"
+)
+
+func TestFetchOrderInfoSuccess(t *testing.T) {
+
+	mockedDB := MockDb(t)
+	defer Ctrl.Finish()
+	messages := []*models.Message{&models.Message{Direction: -1, Instrument: "test", Type: models.NewOrder, OrderID: 1, Price: 100, ShareQuantity: 10, Timestamp: 1, SodOffset: 1}}
```

Comment:
You should use the `MakeMsg` factory method here:
```suggestion
	messages := []*models.Message{MakeMsg(DirectionAsk)}
```
All the default values of the factory correspond to the values you entered here.